mmcv/image/geometric.py

```diff
@@ -91,6 +91,42 @@ def imresize(img,
         h_scale = size[1] / h
         return resized_img, w_scale, h_scale
 
+def resize(img, size, dst=None, interpolation=cv2.INTER_LINEAR):
+    resize_fn = _maybe_process_in_chunks(cv2.resize, size=size, dst = dst, interpolation=interpolation)
+    return resize_fn(img) 
+
+def _maybe_process_in_chunks(process_fn, **kwargs):
+    """
```

Comment:
Please add proper reference if these codes are borrowed from other repos.

Reply:
got you.